docs/src/pages/components/progress/CircularColor.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import Stack from '@material-ui/core/Stack';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+export default function CircularColor() {
+  return (
+    <Stack sx={{ color: 'success.main' }} spacing={2} direction="row">
```

Comment:
"success" is a bit odd to use for something like a pending indicator. In the end, we won't know if that transition is going to be successful or not.

Reply:
ok, changing it